packages/web/src/components/Admin/ExpoJudgingSessionList/ExpoJudgingSessionActionsMenu.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import {
+  Menu,
+  MenuButton,
+  Button,
+  MenuList,
+  MenuItem,
+  MenuItemProps,
+  useClipboard,
+} from '@chakra-ui/react';
+import { BsChevronDown } from 'react-icons/bs';
+import { ExpoJudgingSession } from '@hangar/shared';
+import { openSuccessToast } from '../../utils/CustomToast';
+import { env } from '../../../env';
+
+type ExpoJudgingSessionActionsMenuProps = {
+  expoJudgingSession: ExpoJudgingSession;
+};
+
+const menuItemStyle: MenuItemProps = {
+  py: 3,
+};
+
+export const ExpoJudgingSessionActionsMenu: React.FC<ExpoJudgingSessionActionsMenuProps> = ({
+  expoJudgingSession,
+}) => {
+  const router = useRouter();
```

Comment:
i dont know much about front-end but why are you trying to import a router in a component?

Reply:
Routing can exist at every level of the stack. Here, useRouter provides access to a front-end router; this is usually used to push new URLs to the browsers `history` array, while also rendering new components/views that are subscribed to the new route. In SPAs this allows the browsers `back` `forward` and `refresh` buttons to function as expected when no server based navigation occurs.